homeassistant/components/wallbox/switch.py

```diff
@@ -0,0 +1,88 @@
+"""Home Assistant component for accessing the Wallbox Portal API. The lock component creates a switch entity."""
```

Comment:
`"lock component"` Is that correct?

Reply:
No, I will change that to switch component